viite-backend/viite-main/src/main/scala/fi/liikennevirasto/viite/dao/LinearLocationDAO.scala

```diff
@@ -481,33 +482,36 @@ class LinearLocationDAO extends BaseDAO {
     var closingBracket = s""
     val precision = MaxDistanceForConnectedLinks
 
-    for (i <- 2 until math.min(llLength, 8)) {
+    for (i <- 2 until math.min(llLength, 8)) { // loop through the other than the first two (if any) LLs
```

Comment:
Näitä numeroita 2 ja 8 voisi vähän avata ja mahdollisesti nimetä selkeyden vuoksi?